src/components/Editor.vue

```diff
@@ -5,6 +5,7 @@
     export default {
         props: {
             theme: String,
+            bufferPath: String,
```

Comment:
We don't need to make `Editor` bufferPath "aware", since we can use `CONFIG.get()` in `buffer.js`.